python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -1042,10 +1042,34 @@ def execute_asset_backfill_iteration(
             )
             instance.update_backfill(updated_backfill)
 
+        new_materialized_partitions = (
+            updated_backfill_data.materialized_subset
+            - previous_asset_backfill_data.materialized_subset
+        )
+        new_failed_partitions = (
+            updated_backfill_data.failed_and_downstream_subset
+            - previous_asset_backfill_data.failed_and_downstream_subset
+        )
+        new_requested_partitions = (
+            updated_backfill_data.requested_subset - previous_asset_backfill_data.requested_subset
+        )
         logger.info(
             f"Asset backfill {updated_backfill.backfill_id} completed iteration with status {updated_backfill.status}."
         )
         logger.info(
+            "Backfill iteration summery:\n"
+            f"**Assets materialized since last iteration:**\n {_asset_key_partition_key_iter_to_str(new_materialized_partitions.iterate_asset_partitions()) if new_materialized_partitions.num_partitions_and_non_partitioned_assets > 0 else 'None'}\n"
+            f"**Assets failed since last iteration and their downstream assets:**\n {_asset_key_partition_key_iter_to_str(new_failed_partitions.iterate_asset_partitions()) if new_failed_partitions.num_partitions_and_non_partitioned_assets > 0 else 'None'}\n"
+            f"**Assets requested by this iteration:**\n {_asset_key_partition_key_iter_to_str(new_requested_partitions.iterate_asset_partitions()) if new_requested_partitions.num_partitions_and_non_partitioned_assets > 0 else 'None'}\n"
+        )
+
+        logger.info(
+            "Everything the backfill daemon has done:\n"
+            f"**Materialized assets:**\n {_asset_key_partition_key_iter_to_str(updated_backfill_data.materialized_subset.iterate_asset_partitions()) if updated_backfill_data.materialized_subset.num_partitions_and_non_partitioned_assets > 0 else 'None'}\n"
+            f"**Failed assets and their downstream assets:**\n {_asset_key_partition_key_iter_to_str(updated_backfill_data.failed_and_downstream_subset.iterate_asset_partitions()) if updated_backfill_data.failed_and_downstream_subset.num_partitions_and_non_partitioned_assets > 0 else 'None'}\n"
+            f"**Assets requested or in progress:**\n {_asset_key_partition_key_iter_to_str(updated_backfill_data.requested_subset.iterate_asset_partitions()) if updated_backfill_data.requested_subset.num_partitions_and_non_partitioned_assets > 0 else 'None'}\n"
+        )
+        logger.debug(
```

Comment:
I want to keep the print of the `AssetBackfillData` object since it can be useful for debugging, especially if any spicy window issues come back. Switching it to a debug makes it greyed out in the backfill logs, so it's less distracting

Reply:
👍